docs/3.2. Discovery - Peer to Peer Operation.md

```diff
@@ -51,7 +51,7 @@ When a Node is operating in peer-to-peer mode it MUST additionally advertise the
 | `ver_snd`           | `/senders`                          |
 | `ver_rcv`           | `/receivers`                        |
 
-The value of each of the above SHOULD be an unsigned 8-bit integer initialised to `0`. This integer MUST be incremented and mDNS TXT record updated whenever a change is made to the corresponding HTTP API resource on the Node. The integer MUST wrap back to a value of `0` after reaching a maximum value of `255` (MAX_UINT8_T).
+The value of each of the above SHOULD be an unsigned 8-bit integer initialised to 0. This integer MUST be incremented and mDNS TXT record updated whenever a change is made to the corresponding HTTP API resource on the Node. The integer MUST wrap back to a value of 0 after reaching a maximum value of 255 (MAX_UINT8_T).
```

Comment:
Hmm, this now reads even less like it means the _text representation of_ an unsigned 8-bit integer? [RFC 6763 Section 6.5](https://tools.ietf.org/html/rfc6763#section-6.5) says the value is often ASCII or UTF-8 but can be opaque binary data. This probably needs a rewrite. :-/

Reply:
Suggestion...
```suggestion
The value of each of the above SHOULD be the text representation of an unsigned 8-bit integer initialised to 0. This integer MUST be incremented and mDNS TXT record updated whenever a change is made to the corresponding HTTP API resource on the Node. The integer MUST wrap back to a value of 0 after reaching a maximum value of 255.
```